4.LastFM/js/drawContent.js

```diff
@@ -0,0 +1,148 @@
+var drawSearchPageHandler = function(data) {
+  var unitsBox = document.getElementsByClassName('units-box')[0];
+//var page = document.createDocumentFragment();
+  unitsBox.innerHTML = '';
+  if(+data.results['opensearch:totalResults'] === 0) {
+    return;
+  }
+  drawUnits(unitsBox, 'artist', data.results.artistmatches.artist);
+
+  var pagesNum = data.results['opensearch:totalResults']/
```

Comment:
weird codestyle
what is wrong with `var pagesNum = data.results['opensearch:totalResults'] / data...;`
you are free to have 110symbols